exporters/metrics/src/main/java/com/google/cloud/opentelemetry/metric/MetricExporter.java

```diff
@@ -167,7 +167,12 @@ public CompletableResultCode export(Collection<MetricData> metrics) {
           continue;
       }
       // Update metric descriptors based on configured strategy.
-      metricDescriptorStrategy.exportDescriptors(builder.getDescriptors(), this::exportDescriptor);
+      try {
```

Comment:
This change is also not related to adding a GKE resource.
I think there is some issue with trying to create a metric descriptor for a metric which already exists in Cloud Monitoring, but not sure what is the correct way to handle this.